core/src/com/mygdx/insanerpg/item/Item.java

```diff
@@ -0,0 +1,10 @@
+package com.mygdx.insanerpg.item;
+
+public class Item {
+    private int _sellPrice;
+    private int _buyPrice;
+    private String _name;
+    private String _rarity;
```

Comment:
Not sure if it should be string, look up enums.

Reply:
To do